examples/step-3/doc/results.dox

```diff
@@ -224,3 +224,212 @@ suggestions:
   Again, the difference between two adjacent values goes down by about a
   factor of four, indicating convergence as ${\cal O}(h^2)$.
 </ul>
+<h4>Using the %HDF5 Library to output the solution and additional data</h4>
+%HDF5 is a commonly used format that can be read by many scripting languages (e.g. R or Python). Here are some 
+insights on what is possible.
+<h5> Changing the output from .gnuplot to .h5:</h5>
+     To fully make use of the automation we first need to introduce a private variable for the number of 
+     global refinement steps <code>unsigned int n_refinement_steps</code>, which will be used for the output filename. 
+     In <code>make_grid()</code> we replace <code>triangulation.refine_global(5);</code> with
+     @code
+     n_refinement_steps = 5;
+     triangulation.refine_global(n_refinement_steps);
+     @endcode
+     So far, the internal bindings for %HDF5 in deal.II only support parallel output with MPI.
```

Comment:
```suggestion
     In the deal.II library there are two different %HDF5 bindings, one in the HDF5 namespace and another one in DataOut.
     Although the HDF5 deal.II binding supports both, serial and MPI, the %HDF5 DataOut binding only support parallel output.
```
Note that I used  HDF5 if I refer to the HDF5 namespace and %HDF5 if I refer to the library.